query_server/query/src/metadata.rs

```diff
@@ -166,6 +178,179 @@ impl MetaData for LocalCatalogMeta {
         Ok(())
     }
 
+    fn show_databases(&self) -> Result<Output> {
+        let dbs = self.engine.list_databases();
+
+        match dbs {
+            Ok(databases) => {
+                let schema = Arc::new(Schema::new(vec![Field::new(
+                    "Database",
+                    DataType::Utf8,
+                    false,
+                )]));
+
+                let batch =
+                    RecordBatch::try_new(schema, vec![Arc::new(StringArray::from(databases))])
+                        .unwrap();
+
+                let batches = vec![Arc::new(batch)];
+
+                Ok(Output::StreamData(stream_from_batches(batches)))
+            }
+            Err(_err) => Err(MetadataError::InternalError {}),
```

Comment:
Dropping the exception directly should not be the best way, because we need to understand and trace the source of the exception

Reply:
nice hint